### PR TITLE
64 apply new styling concept

### LINKED ISSUE #64: Apply CSS concept
As programmers, we now want to apply the concept to our existing code and frontend.

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] Existing concept was applied
- [ ] There is no inline CSS in our code anymore
- [x] requires #62 & #63

### PR DESCRIPTION
**Pull request by WolfgangDaniel**
closes #64 

## Changes
The Styling of the website was inconsistent and without a corporate design. This issue implemented a new corporate design and applied the new styling concept. Although some minor changes to the frontend design were made this issue does not included a structured redesign of the website. 

## Screenshots
Before:
grafik
grafik

After: 
grafik
grafik

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

**Comment by kej-jay** [comment]
Since you are using croco the parser test is not running through anymore. Please fix that and I will look at the code.

File: client/src/components/pages/RobotFile/RobotFile.jsx [hunk 1]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { Layout } from 'antd';
 import HeaderNavbar from '../../content/HeaderNavbar/HeaderNavbar';
 import { Typography } from 'antd';
```
Comment (kej-jay) [inline client/src/components/pages/RobotFile/RobotFile.jsx:4]: Can Typography and Layout be imported in the same statement?

File: client/src/layout/corporateDesign.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* changes to the variables here also have to be made in corporateDesign.js !!! */
```
Comment (kej-jay) [inline client/src/layout/corporateDesign.css:1]: Please avoid the usage of three exclamation marks (!!!)

File: client/src/layout/corporateDesign.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// changes to the variables here also have to be made in corporateDesign.css !!!
```
Comment (kej-jay) [inline client/src/layout/corporateDesign.js:1]: Same as above

**Review by kej-jay — CHANGES_REQUESTED** [review]
Some minor changes, please also look at the client-side test (see my comment previously)

File: client/package.json [hunk 4]
```diff
@@ -21,9 +24,9 @@
     "xml-js": "^1.6.11"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
```
Comment (SanJSp) [inline client/package.json:29]: Does `npm test` still work like before?

**Review by SanJSp — APPROVED** [review]
apart from that one question, it looks fine :)

**Comment by SanJSp** [comment]
> 
> 
> Since you are using croco the parser test is not running through anymore. Please fix that and I will look at the code.
> EDIT: The Testing error was on the dev branch as well, I have fixed it for my branch. Please have a look at how I did it (just some import/export stuff) and check, if that's the only cause for your tests to fail too. I will look at the rest f the code soon

regarding that, we should really think about a github workflow for tests. We should not have to run tests of the PR on our machines

**Comment by kej-jay** [comment]
> 
> 
> > Since you are using croco the parser test is not running through anymore. Please fix that and I will look at the code.
> > EDIT: The Testing error was on the dev branch as well, I have fixed it for my branch. Please have a look at how I did it (just some import/export stuff) and check, if that's the only cause for your tests to fail too. I will look at the rest f the code soon
> 
> regarding that, we should really think about a github workflow for tests. We should not have to run tests of the PR on our machines

Das hat der neue PO schon auf der Liste stehen ^^

File: client/src/components/pages/Error/Error.jsx [hunk 5]
```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import HeaderNavbar from '../../content/HeaderNavbar/HeaderNavbar';
 import { Layout, Typography } from 'antd';
```
[the remark this answers is not in the document]
Comment (WolfgangDaniel) [reply client/src/components/pages/Error/Error.jsx:3]: done

File: client/src/components/pages/RobotFile/RobotFile.jsx (hunk 1 above)
Comment (WolfgangDaniel) [reply client/src/components/pages/RobotFile/RobotFile.jsx:4]: yes, good point. Done.

File: client/src/layout/corporateDesign.css (hunk 2 above)
Comment (WolfgangDaniel) [reply client/src/layout/corporateDesign.css:1]: fair haha changed it to one exclamation mark

File: client/src/layout/corporateDesign.js (hunk 3 above)
Comment (WolfgangDaniel) [reply client/src/layout/corporateDesign.js:1]: same here

File: client/package.json (hunk 4 above)
Comment (WolfgangDaniel) [reply client/package.json:29]: it should

**Comment by WolfgangDaniel** [comment]
> 
> 
> Since you are using croco the parser test is not running through anymore. Please fix that and I will look at the code.
> EDIT: The Testing error was on the dev branch as well, I have fixed it for my branch. Please have a look at how I did it (just some import/export stuff) and check, if that's the only cause for your tests to fail too. I will look at the rest f the code soon

I will try to fix it. To which branch are you referring to with "my branch"?

**Comment by kej-jay** [comment]
> > Since you are using croco the parser test is not running through anymore. Please fix that and I will look at the code.
> > EDIT: The Testing error was on the dev branch as well, I have fixed it for my branch. Please have a look at how I did it (just some import/export stuff) and check, if that's the only cause for your tests to fail too. I will look at the rest f the code soon
> 
> I will try to fix it. To which branch are you referring to with "my branch"?

I think I was referencing the 87_backend... branch

**Review by kej-jay — APPROVED** [review]
The test should be fixed after merging that other branch I was referring too, so from my side everything should be good to go.

**Review by SanJSp — APPROVED** [review]

**Pull request merged**